.gitignore

```diff
@@ -27,4 +27,6 @@ yarn.lock
 package-lock.json
 
 # IntelliJ
-.idea/
\ No newline at end of file
+.idea/
+src/firebase/firebaseTest.js
+src/firebase/firebaseTestConfig.js
```

Comment:
I think these two files should be staged. If there is any sensible data it can be referenced as an environment variable in `.env.development.local`